components/brave_wallet/browser/test/BUILD.gn

```diff
@@ -152,7 +152,12 @@ source_set("brave_wallet_unit_tests") {
   if (enable_orchard) {
     sources += [
       "//brave/components/brave_wallet/browser/internal/hd_key_zip32_unittest.cc",
+      "//brave/components/brave_wallet/browser/internal/orchard_block_scanner_unittest.cc",
       "//brave/components/brave_wallet/browser/internal/orchard_bundle_manager_unittest.cc",
+      "//brave/components/brave_wallet/browser/zcash/zcash_orchard_storage_unittest.cc",
+      "//brave/components/brave_wallet/browser/zcash/zcash_shield_sync_service_unittest.cc",
+      "//brave/components/brave_wallet/browser/zcash/zcash_test_utils.cc",
+      "//brave/components/brave_wallet/browser/zcash/zcash_test_utils.h",
```

Comment:
these two most likely should be in `test_support` set

Reply:
Done